.gas-report

```diff
@@ -1,12 +1,12 @@
-╭──────────────────────────────────────────┬─────────────────┬───────┬────────┬───────┬─────────╮
-│ src/GtcRadGrant.sol:GtcRadGrant contract ┆                 ┆       ┆        ┆       ┆         │
-╞══════════════════════════════════════════╪═════════════════╪═══════╪════════╪═══════╪═════════╡
-│ Deployment Cost                          ┆ Deployment Size ┆       ┆        ┆       ┆         │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ 372494                                   ┆ 2084            ┆       ┆        ┆       ┆         │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ Function Name                            ┆ min             ┆ avg   ┆ median ┆ max   ┆ # calls │
-├╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
-│ grant                                    ┆ 305             ┆ 61215 ┆ 80452  ┆ 83652 ┆ 4       │
-╰──────────────────────────────────────────┴─────────────────┴───────┴────────┴───────┴─────────╯
+╭──────────────────────────────────────────┬─────────────────┬───────┬────────┬────────┬─────────╮
```

Comment:
we should also double check if our homegrown solution is still needed. it wouldn't surprise me if foundry has this built-in now

Reply:
Oh right. Yeah I'm going to spend some time this week updating our template and reading into the latest foundry features. They just released a bunch of cool new features.